_episodes/14-live.md

```diff
@@ -123,6 +123,8 @@ Some instructors use printouts of the lesson material during teaching. Others us
 9. **Embrace mistakes.** No matter how well prepared you are, you will make mistakes. This is OK! Use these opportunities to do [error framing]({{ page.root }}{% link _episodes/09-mindset.md %}#errors-are-essential-to-learning) and to help your learners learn the art of troubleshooting.
 10. **Have fun!** It's OK to use humor and improvisation to liven up the workshop. This becomes easier when you are more familiar with the material, and more relaxed. Start small, even just saying 'that was fun' after something worked well is a good start.
 
+Read more about these tips by reading [this paper][live-coding-tips-paper]
```

Comment:
It's best to avoid link descriptions that say "this X", so here's a suggestion for a more descriptive link:

```suggestion
Read more in [Ten quick tips for teaching with participatory live-coding][live-coding-tips-paper]
```

Reply:
Great idea!  Was co-teaching while trying to get this added and didn't quite have the words at the time.